src/entity/Guild/Guild.ts

```diff
@@ -1,5 +1,6 @@
 import { Entity, PrimaryColumn, Column, OneToOne, OneToMany } from 'typeorm'
 import { GuildSettings, GuildSuggestion, GuildSupportTicket } from '.'
```

Comment:
Add the GuildUser class to the index.ts file like
```
export * from './GuildUser'
```

Then add this import in the existing import on line 2.